model/feature-flag/registry.yaml

```diff
@@ -9,26 +9,45 @@ groups:
       - id: feature_flag.key
         type: string
         stability: experimental
-        brief: The unique identifier of the feature flag.
+        brief: The lookup key of the feature flag.
         examples: ["logo-color"]
-      - id: feature_flag.provider_name
+      - id: feature_flag.provider.id
         type: string
         stability: experimental
-        brief: The name of the service provider that performs the flag evaluation.
+        brief: Any string which identifies the feature flag provider being used such as its name.
         examples: ["Flag Manager"]
       - id: feature_flag.variant
         type: string
         stability: experimental
         examples: ["red", "true", "on"]
         brief: >
-          SHOULD be a semantic identifier for a value. If one is unavailable, a
-          stringified version of the value can be used.
+          A semantic identifier for an evaluated flag value.
         note: |-
           A semantic identifier, commonly referred to as a variant, provides a means
           for referring to a value without including the value itself. This can
           provide additional context for understanding the meaning behind a value.
           For example, the variant `red` maybe be used for the value `#c05543`.
-
-          A stringified version of the value can be used in situations where a
-          semantic identifier is unavailable. String representation of the value
-          should be determined by the implementer.
+      - id: feature_flag.value
+        type: string
+        stability: experimental
+        examples: ["#ff0000", "1", "true"]
+        brief: >
+          The evaluated value of the feature flag.
+      - id: feature_flag.context.id
+        type: string
+        stability: experimental
+        examples: ["5157782b-2203-4c80-a857-dbbd5e7761db"]
+        brief: >
+          The unique identifier for the flag evaluation context. For example, the targeting key.
+      - id: feature_flag.version
+        type: string
+        stability: experimental
+        examples: ["1", "01ABCDEF"]
+        brief: >
+          The version of the ruleset used during the evaluation. This may be any stable value which uniquely identifies the ruleset.
+      - id: feature_flag.flag_set.id
```

Comment:
nit: consider `feature_flag.set.id` - I understand 'flag set' is existing terminology, but `feature_flag.set.id` is still pretty clear and close, but shorter

Reply:
I'm in favor of this change. 👍